homeassistant/components/automation/__init__.py

```diff
@@ -732,7 +722,7 @@ def started_action() -> None:
                     translation_placeholders={
                         "service": f"{err.domain}.{err.service}",
                         "entity_id": self.entity_id,
-                        "name": self.name or self.entity_id,
+                        "name": self._attr_name or self.entity_id,
```

Comment:
Not that it really matters, because the `self.name` property just returns `self._attr_name` anyway, but why replace it with the direct attribute reference here? `self.name` is used in some other methods below.

Reply:
In the base entity name is a cached property so it will only every be calculated once an than saved in __dict__ so no function call ever has to be made again once is saved once

Reply:
Thanks